connector/datadogconnector/README.md

```diff
@@ -104,4 +104,9 @@ service:
 ```
 </tr></table>
 
-Here we have two traces pipelines that ingest the same data but one is being sampled. The one that is sampled has its data sent to the datadog backend for you to see the sampled subset of the total traces sent across. The other non-sampled pipeline of traces sends its data to the metrics pipeline to be used in the APM stats. This unsampled pipeline gives the full picture of how much data the application emits in traces. 
+Here we have two traces pipelines that ingest the same data but one is being sampled. The one that is sampled has its data sent to the datadog backend for you to see the sampled subset of the total traces sent across. The other non-sampled pipeline of traces sends its data to the metrics pipeline to be used in the APM stats. This unsampled pipeline gives the full picture of how much data the application emits in traces.
+
+## Feature Gate for Performance
+
```

Comment:
We should explicitly say that the feature gate needs to be enabled on all Collectors that use the Datadog connector and all Collectors that use the Datadog exporter. On a gateway deploy, if the connector is running on a different binary from the exporter and you enable the feature gate only on one of them, it will silently fail